ctf/setup.py

```diff
@@ -49,3 +54,8 @@ def build_challenges():
                     db.session.commit()
                 except IntegrityError:
                     db.session.rollback()
+                    challenge = Challenge.query.filter_by(
+                            title=problem['title'])
+                    problem_dict.update({'id': challenge.first().id})
```

Comment:
The reason I hesitate against that is because we have two cases in this file:

1. We are adding a challenge for the first time. In this case, the function takes the configuration given to us by the json file and converts it into the configuration required for building the SQL object

2. We are updating a challenge. In this case, we already go through the function `build_problem_options` to convert the json to the proper format, so all we need to do is add the current id to that field and update with it.